XenAdmin/Dialogs/AddVGPUDialog.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;

```

Comment:
The following `using` directives are not needed: 
System.Diagnostics, System.Drawing, System.ComponentModel, System.Data, System.Text, System.Threading.Tasks, 
System.Windows.Forms, XenAdmin.Actions, XenAdmin.Properties